pkg/lib/kitfile/local-storage.go

```diff
@@ -89,64 +90,81 @@ func saveConfig(ctx context.Context, localRepo local.LocalRepo, kitfile *artifac
 	return desc, nil
 }
 
+// SaveKitfileLayers saves the kitfile layers with optional compression and returns a list of descriptors.
```

Comment:
Nitpick: the first word should match the function name (lowercase here, since it's not exported)
```suggestion
// saveKitfileLayers saves the kitfile layers with optional compression and returns a list of descriptors.
```